src/styles/blocks/calendar.scss

```
@@ -0,0 +1,38 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  justify-content: flex-start;
+  align-items: center;
+  width: 713px;
+  :first-child {
+    margin-left: 612px;
+  }
+}
+
+@for $i from 1 through 31 {
+  .day#{$i}:before {
+    content: "#{$i}";
+  }
+}
+
+.calendar__day {
+  display: flex;
+  align-items: center;
```

```suggestion
  align-items: center;
  justify-content: center;
```
можно сделать `placeholder` и добавить в файл `_extend`